internal/provider/bootstrap.go

```diff
@@ -48,9 +48,11 @@ func Bootstrap(e *pluggable.Event) pluggable.EventResponse {
 	}
 	// TODO: this belong to a systemd service that is started instead
 
-	tokenNotDefined := (providerConfig.Kairos != nil && providerConfig.Kairos.NetworkToken == "")
+	kairosBlockisDefined := providerConfig.P2P != nil
```

Comment:
Should we rename this to `p2pBlockDefined` now?

Reply:
Good point, fixed!